x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/40_source_mode_setting.yml

```diff
@@ -198,174 +194,56 @@ create an index with synthetic source mode and time series index mode without se
       indices.get_settings:
         index: "test_synthetic_time_series"
   - match: { test_synthetic_time_series.settings.index.mode: time_series }
-
-  - do:
-      indices.get_mapping:
-        index: test_synthetic_time_series
-
-  - match: { test_synthetic_time_series.mappings._source.mode: synthetic }
+  - match: { test_synthetic_time_series.settings.index.mapping.source.mode: synthetic }
 
 ---
 create an index with stored source mode:
   - do:
       indices.create:
         index: test_stored_default
-        body:
-          mappings:
-            _source:
-              mode: stored
-
-  - do:
-      indices.get_mapping:
-        index: test_stored_default
-
-  - match: { test_stored_default.mappings._source.mode: stored }
-
----
-override stored to synthetic source mode:
-  - do:
-      indices.create:
-        index: test_stored_override
         body:
           settings:
             index:
-              mapping.source.mode: synthetic
-          mappings:
-            _source:
-              mode: stored
+              mapping:
+                source:
+                  mode: stored
 
   - do:
-      indices.get_mapping:
-        index: test_stored_override
-
-  - match: { test_stored_override.mappings._source.mode: synthetic }
-
----
-override stored to disabled source mode:
```

Comment:
Tests that both that test overwrite behaviour when source mode index setting and _source.mode mapping attribute are specified are no longer relevant with this change.